### PR TITLE
Moved SystTools from icaruscode so we share this with SBND

### PR DESCRIPTION
**Pull request by jedori0228**
Moving `icarsucode/fcl/caf/SystTools` to `sbncode/SBNEventWeight/jobs/SystTools`. The universes are already thrown in fcl, so it is important to have same configs between SBND and ICARUS. I am creating another PR in `icaruscode` where I removed the directory.

### CODE REVIEW

**Comment by jedori0228** [comment]
https://github.com/SBNSoftware/icaruscode/pull/697 is created.

File: sbncode/SBNEventWeight/jobs/SystTools/configs/config_All_icarus_v2.fcl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#include "config_GENIEReWeightMultisigma_icarus_v2.fcl"
+#include "config_GENIEReWeightMultisim_icarus_v2.fcl"
+#include "config_MINERvA_E2p2h_icarus_v1.fcl"
+#include "config_MINERvAq0q3Tune_icarus_v1.fcl"
+#include "config_MiscInteractionSysts_icarus_v1.fcl"
+#include "config_NOvAStyleNonResPion_icarus_v1.fcl"
```
Comment (henrylay97) [inline sbncode/SBNEventWeight/jobs/SystTools/configs/config_All_icarus_v2.fcl:6]: Do we want to change the file names to reflect `sbn` status?

File: sbncode/SBNEventWeight/jobs/SystTools/configs/config_GENIEReWeightMultisigma_icarus_v2.fcl [hunk 2]
```diff
@@ -0,0 +1,348 @@
+GENIEReWeight_Multisigma_Tool_Config: {
+
+    tool_type: "GENIEReWeight"
+
+    # Updated 25 July 2023 by S. Gardiner
+    instance_name: "ICARUS_v2_multisigma"
```
Comment (henrylay97) [inline sbncode/SBNEventWeight/jobs/SystTools/configs/config_GENIEReWeightMultisigma_icarus_v2.fcl:6]: And instance names too?

**Review by henrylay97 — COMMENTED** [review]
Hi @jedori0228  thank you so much for doing that :D 

This looks great to me. I have one query which is this:

Do we want to rename the files & instance names to be `sbn_v1`  and similar (rather than ICAURS type names). I appreciate the actual config isn't changing and this will affect ICARUS analysis scripts which would need to adopt a new naming convention for reading the weights. I do feel it might avoid future confusion though.

What are your thoughts?

**Comment by jedori0228** [comment]
@henrylay97 Thanks Henry, I just updated the filenames and strings in fhicls.

**Review by PetrilloAtWork — COMMENTED** [review]
It looks good to me.
I would like somebody else to look at this too, though.
But ping me in case a faster approval is needed, and I will turn this into an approval.

**Comment by fjnicolas** [comment]
trigger build SBNSoftware/icaruscode#697 SBNSoftware/sbndcode#417

**Pull request merged**